cli/pkg/kctrl/cmd/package/available/get.go

```diff
@@ -30,11 +30,12 @@ type GetOptions struct {
 
 	ValuesSchema bool
 
+	binaryName        string
 	positionalNameArg bool
```

Comment:
should we turn these into a struct (it's used all over) and it gets pretty noisy to pass them around as individual args.

Reply:
Yep, I was thinking of doing the same, I was a little hesitant because then we would have opts inside other opts (GetOptions, ListOptions, etc,.), but I think it makes sense have a struct, maybe we can name it something else.